server/handle_capabilities_test.go

```diff
@@ -120,12 +125,169 @@ func TestHandleCapabilities(t *testing.T) {
 				},
 			},
 		},
+		{
+			handler:    server.HandleCapabilities{},
+			uri:        "http://localhost:8080/capabilities",
+			uriPattern: "/capabilities",
+			reqMethod:  "GET",
+			expected: server.Capabilities{
+				Version: serverVersion,
+				Maps: []server.CapabilitiesMap{
+					{
+						Name:         "test-map",
+						Attribution:  "test attribution",
+						Center:       [3]float64{1.0, 2.0, 3.0},
+						Bounds:       [4]float64{-180.0, -85.0511, 180.0, 85.0511},
```

Comment:
Might be good to use this package variable:
https://github.com/terranodo/tegola/blob/v0.5.0/projections.go#L10

Reply:
Would certainly make it cleaner to read.